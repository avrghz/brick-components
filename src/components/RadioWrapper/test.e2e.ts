import { newE2EPage, E2EPage, E2EElement } from '@stencil/core/testing'

describe('<bk-radio-wrapper/>', () => {
    let page: E2EPage
    let component: E2EElement

    beforeEach(async () => {
        page = await newE2EPage()
        await page.setContent(`<bk-radio-wrapper label="Option">
             <input type="radio" aria-hidden="false" value="1" name="options" />
        </bk-radio-wrapper>`)
        component = await page.find('bk-radio-wrapper')
    })

    it('should render', async () => {
        expect(component).toMatchSnapshot('snapshot__basic')
    })
})
