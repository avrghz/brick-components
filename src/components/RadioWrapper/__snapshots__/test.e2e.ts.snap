// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<bk-radio-wrapper/> should render: snapshot__basic 1`] = `
<bk-radio-wrapper class="hydrated sc-bk-radio-wrapper-h sc-bk-radio-wrapper-s" label="Option">
  <!---->
  <label class="bk-radio sc-bk-radio-wrapper sc-bk-radio-wrapper-s">
    <input aria-hidden="false" name="options" type="radio" value="1">
    <span class="bk-radio__input sc-bk-radio-wrapper">
      <span class="bk-radio__inner sc-bk-radio-wrapper"></span>
    </span>
    <span class="bk-radio__label sc-bk-radio-wrapper">
      Option
    </span>
  </label>
</bk-radio-wrapper>
`;
